patch/0001-hwmon-supporting-emc2302.patch

```diff
@@ -0,0 +1,38 @@
+From 291cfd2e11a31164aaf9eeff38447fc553c74dbd Mon Sep 17 00:00:00 2001
+From: Natarajan Subbiramani <natarajan.subbiramani.ext@nokia.com>
+Date: Mon, 20 Mar 2023 18:22:33 +0000
+Subject: [PATCH] Adding EMC2302 support
```

Comment:
Please follow [How to write a good commit message](https://cbea.ms/git-commit/).